Packs/ImpossibleTraveler/Playbooks/playbook-Impossible_Traveller.yml

```diff
@@ -1,3365 +1,3046 @@
+- key: WhitelistedIPs
+  value: {}
+  required: false
+  description: CSV of IP addresses that are allowed to be used across long distances.
+- key: AutomaticallyBlockIPs
+  value:
+    simple: 'False'
+  required: false
+  description: Whether to automatically block the source IPs that the login originated
+    from. Can be False or True.
+- key: DefaultMapLink
+  value:
+    simple: https://bing.com/maps/default.aspx?rtp=pos.SOURCE~pos.DESTINATION
+  required: false
+  description: The default link from which to create a travel map. The "SOURCE" and
+    "DESTINATION" words are replaced with the previous coordinates and current coordinates
+    of the traveler, respectively.
+- key: AutomaticallyDisableUser
+  value:
+    simple: 'False'
+  required: false
+  description: Whether to automatically disable the impossible traveler account using
+    Active Directory.
+- key: ContactUserManager
+  value:
+    simple: 'False'
+  required: false
+  description: Whether to ask the user manager for the legitimacy of the login events,
+    in case of an alleged impossible traveler.
+outputs:
+- contextPath: Account.Email.Address
+  description: The email address object associated with the Account
+  type: string
+- contextPath: DBotScore
+  description: Indicator, Score, Type, Vendor
+  type: unknown
+- contextPath: Account.ID
+  description: The unique Account DN (Distinguished Name)
+  type: string
+- contextPath: Account.Username
+  description: The Account username
+  type: string
+- contextPath: Account.Email
+  description: The email address associated with the Account
+- contextPath: Account.Type
+  description: Type of the Account entity
+  type: string
+- contextPath: Account.Groups
+  description: The groups the Account is a part of
+- contextPath: Account
+  description: Account object
+  type: unknown
+- contextPath: Account.DisplayName
+  description: The Account display name
+  type: string
+- contextPath: Account.Manager
+  description: The Account's manager
+  type: string
+- contextPath: DBotScore.Indicator
+  description: The indicator value
+  type: string
+- contextPath: DBotScore.Type
+  description: The indicator's type
+  type: string
+- contextPath: DBotScore.Vendor
+  description: The indicator's vendor
+  type: string
+- contextPath: DBotScore.Score
+  description: The indicator's score
+  type: number
+- contextPath: IP
+  description: The IP objects
+  type: unknown
+- contextPath: Endpoint
+  description: The Endpoint's object
+  type: unknown
+- contextPath: Endpoint.Hostname
+  description: The hostname to enrich
+  type: string
+- contextPath: Endpoint.OS
+  description: Endpoint OS
+  type: string
+- contextPath: Endpoint.IP
+  description: List of endpoint IP addresses
+- contextPath: Endpoint.MAC
+  description: List of endpoint MAC addresses
+- contextPath: Endpoint.Domain
+  description: Endpoint domain name
+  type: string
+sourceplaybookid: Impossible Traveler

```

Comment:
did you not run format on this playbook?
remove that field

Reply:
I ran format (hence the huge change all over the playbook), but apparently it doesn't remove it, so I'll remove manually